app/src/main/java/com/kks/nimblesurveyjetpackcompose/model/SurveyAnswer.kt

```diff
@@ -4,12 +4,13 @@ import com.kks.nimblesurveyjetpackcompose.model.request.SurveyAnswerRequest
 
 data class SurveyAnswer(
     val id: String,
-    var text: String,
+    val text: String,
     val displayOrder: Int,
-    var selected: Boolean = false
+    var selected: Boolean = false,
+    var answer: String
```

Comment:
if we set a default value for `answer` we don't need to update `IncludedResponse.toSurveyAnswer()` and the previews anymore 😉 
```suggestion
    var answer: String = ""
```